kubebuilder.mk

```diff
@@ -27,6 +27,8 @@ endif
 # enable Go modules
 export GO111MODULE=on
 
+CONTROLLER_GEN=GOFLAGS=-mod=vendor go run sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
Can we put quotes around this, unless it is a Makefile-ism

Reply:
It's a Makefile-ism